src/scripts/batch/hit_counts.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Using user query strings from the input stream,
+ * this script gets the number pathways and interactions (SIF edges).
+ * Results will be the same as or close (see comments below) to
+ * what one could get at apps.pathwaycommons.org if settings and queries are same.
+ *
+ * This script simply calls the app-ui server-side modules
+ * (does not start the nore server nor web app),
+ * which, in order, search for pathways and inferred simple
+ * interactions in the Pathway Commons backend (PC2 and Sifgraph services).
+ *
+ * @author Igor Rodchenkov
+ */
+
```

Comment:
Suggestion is to insert `global.fetch = require('node-fetch');` as in server/index.js  instead of below.

Reply:
done